pkg/controller/apiserver/apiserver_controller.go

```diff
@@ -284,25 +290,78 @@ func (r *ReconcileAPIServer) Reconcile(ctx context.Context, request reconcile.Re
 
 	// Render the desired objects from the CRD and create or update them.
 	reqLogger.V(3).Info("rendering components")
+	var components []render.Component
 	component, err := render.APIServer(k8sapi.Endpoint, network, false, managementCluster, managementClusterConnection, amazon, tlsSecret, pullSecrets, r.provider == operatorv1.ProviderOpenShift,
 		tunnelCASecret, r.clusterDomain)
 	if err != nil {
 		log.Error(err, "Error rendering APIServer")
 		r.status.SetDegraded("Error rendering APIServer", err.Error())
 		return reconcile.Result{}, err
 	}
+	components = append(components, component)
+
+	if variant == operatorv1.TigeraSecureEnterprise {
+
+		var serverCertSecret *v1.Secret
+		if network.CertificateManagement == nil {
+			serverCertSecret, err = utils.ValidateCertPair(r.client,
+				rmeta.OperatorNamespace(),
+				render.PacketCaptureCertSecret,
+				v1.TLSPrivateKeyKey,
+				v1.TLSCertKey,
+			)
+			if err != nil {
+				log.Error(err, fmt.Sprintf("failed to retrieve / validate %s", render.PacketCaptureCertSecret))
+				r.status.SetDegraded(fmt.Sprintf("failed to retrieve / validate  %s", render.PacketCaptureCertSecret), err.Error())
```

Comment:
Nit, but this is a user facing message so please capitalise the first let of the sentence.